feral-lambda-runtime/src/main/scala/feral/lambda/runtime/LambdaRuntimeEnv.scala

```diff
@@ -0,0 +1,70 @@
+package feral.lambda.runtime
+
+import cats.{ApplicativeError, Functor}
+import cats.effect.kernel.Sync
+import cats.effect.std.Env
+import cats.syntax.all._
+
+trait LambdaRuntimeEnv[F[_]] {
+  def handler: F[String]
+  def region: F[String]
+  def executionEnv: F[String]
+  def lambdaFunctionName: F[String]
+  def lambdaFunctionMemorySize: F[Int]
+  def lambdaFunctionVersion: F[String]
+  def lambdaLogGroupName: F[String]
+  def lambdaLogStreamName: F[String]
+  def accessKeyId: F[String]
+  def secretAccessKey: F[String]
+  def lambdaRuntimeApi: F[String]
+  def lambdaTaskRoot: F[String]
+  def lambdaRuntimeDir: F[String]
+  def timezone: F[String]
+}
+
+object LambdaRuntimeEnv {
+  final val HANDLER = "_HANDLER"
+  final val AWS_REGION = "AWS_REGION"
+  final val AWS_EXECUTION_ENV = "AWS_EXECUTION_ENV"
+  final val AWS_LAMBDA_FUNCTION_NAME = "AWS_LAMBDA_FUNCTION_NAME"
+  final val AWS_LAMBDA_FUNCTION_MEMORY_SIZE = "AWS_LAMBDA_FUNCTION_MEMORY_SIZE"
+  final val AWS_LAMBDA_FUNCTION_VERSION = "AWS_LAMBDA_FUNCTION_VERSION"
+  final val AWS_LAMBDA_LOG_GROUP_NAME = "AWS_LAMBDA_LOG_GROUP_NAME"
+  final val AWS_LAMBDA_LOG_STREAM_NAME = "AWS_LAMBDA_LOG_STREAM_NAME"
+  final val AWS_ACCESS_KEY_ID = "AWS_ACCESS_KEY_ID"
+  final val AWS_SECRET_ACCESS_KEY = "AWS_SECRET_ACCESS_KEY"
+  final val AWS_LAMBDA_RUNTIME_API = "AWS_LAMBDA_RUNTIME_API"
+  final val LAMBDA_TASK_ROOT = "LAMBDA_TASK_ROOT"
+  final val LAMBDA_RUNTIME_DIR = "LAMBDA_RUNTIME_DIR"
+  final val TZ = "TZ"
+
+  def apply[F[_]: Functor](env: Env[F]): LambdaRuntimeEnv[F] = new LambdaRuntimeEnv[F] { // should be error effect?
+    override def handler: F[String] = env.get(HANDLER).map(_.get)
```

Comment:
We should avoid `.get`, instead something like this with `MonadThrow[F]`.

```suggestion
    def handler: F[String] = env.get(HANDLER).flatMap(_.liftTo(new NoSuchElementException(HANDLER))
```